core/src/main/web/app/helpers/helper.js

```diff
@@ -205,6 +205,22 @@
       getBaseUrl: function() {
         return bkUtils.getBaseUrl();
       },
+      openNotebookInNewWindow: function (notebookUri, uriType, readOnly, format) {
+
+        var path = "/open?uri=" + window.encodeURIComponent(notebookUri);
```

Comment:
Since $httpParamSerializer will do the encoding, you can use it for uri param also.